### ISSUE TITLE
A11y: Introduce Alternative text to the Message Search control

### ISSUE THREAD

**Issue by daniellekirkwood** [vector-im/element-web#24921]
In the Room, users can click the search icon to search messages. There's no alternative text here and therefore is not usable for screen readers.

<img width="453" alt="Screenshot 2023-03-24 at 11 41 39" src="https://user-images.githubusercontent.com/89144281/227512147-9601b560-965d-48ee-ab59-f260a1f3c551.png">

Please ask @daniellekirkwood about timesheeting

**Comment by t3chguy** [comment]
![image](https://user-images.githubusercontent.com/2403652/227512735-6069093f-76d8-4e7a-99bd-60509eab18be.png)

The header button to reach this state has a label - this issue is about the search action button which can also be triggered by Enter in the text field

![image](https://user-images.githubusercontent.com/2403652/227512827-eb715b2b-c017-4e89-b123-ffc7c06668af.png)

![image](https://user-images.githubusercontent.com/2403652/227512866-d145939c-db81-44e0-97ce-8bd68d21f49b.png)

The X button is also unlabelled
